### PR TITLE
Update some readme's

### PR DESCRIPTION
**Pull request by anandkumarpatel**
* update link to IAM user info
* fix trailing spaces

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/2031)

### CODE REVIEW

File: docs/aws.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ Kops by default will assume that the NS records created above are publicly avail
 Note: There is a DNS flag that can be configured if you plan on using private DNS records
 
 ```
-kops create cluster --dns private $NAME
+kops create cluster --dns-zone private $NAME
```
Comment (justinsb) [inline docs/aws.md]: It is actually `--dns`

File: docs/aws.md [hunk 2]
```diff
@@ -283,12 +283,6 @@ kubectl get nodes
 
 You will see a list of nodes that should match the `--zones` flag defined earlier. This is a great sign that your Kubernetes cluster is online and working.
 
-Also kops ships with a handy validation tool that can be ran to ensure your cluster is working as expected.
-
-```bash
-kops validate cluster
```
Comment (justinsb) [inline docs/aws.md]: This does ship with kops

File: docs/run_in_existing_vpc.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ export CLUSTER_NAME=<sharedvpc.mydomain.com>
 export VPC_ID=vpc-12345678 # replace with your VPC id
 export NETWORK_CIDR=10.100.0.0/16 # replace with the cidr for the VPC ${VPC_ID}
 
-kops create cluster --zones=us-east-1b --name=${CLUSTER_NAME} --vpc=${VPC_ID}
+kops create cluster --zones=us-east-1b --name=${CLUSTER_NAME} --vpc=${VPC_ID} --network-cidr=${NETWORK_CIDR}
```
Comment (justinsb) [inline docs/run_in_existing_vpc.md]: This is now optional

**Comment by justinsb** [comment]
The first two changes look good, but I think you're running an older version of kops.  At least the `--dns` flag should be in 1.5.1

**Comment by anandkumarpatel** [comment]
ahh I see, I am using kops that ships with brew. Will update and try again.

**Comment by justinsb** [comment]
@anandkumarpatel do you want to pursue this PR?  Might be easier to close & submit a new one... 

PS Thank you for contributing either way :-)

**Comment by anandkumarpatel** [comment]
@justinsb just merged with master and it looks like most of my changes were already fixed :) I just have a small nit I updated now.

**Comment by yissachar** [comment]
@k8s-bot ok to test

**Pull request merged**